### DISCUSSION TITLE
Welcome to WingetUI Discussions!

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#245] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Spacellary** [comment]
"It is I, Space!"
I just love the idea of a nicely designed unified GUI for Winget, Scoop and maybe one day Chocolatey. Not particularly a developer or system admin, I just like seeing my updates in a convenient list.

I'm excited for the future of package management in Windows, I think anything that makes it more accessible is a positive! Though I know a lot of people just prefer the reliability of cli, it's good to have options. WingetUI is by far the best project I've seen on that regard, and I do appreciate a lot all the work you put into it and your other projects of course!

**Comment by gidano** [comment]
Hi, I am gidano, from Hungary.
I have been using a computer since the Commode 64 - IBM XT line and I am very interested! Nowadays I'm translating Android (and some PC) applications in my spare time. I'm not a professional, I can't program, although I use computers in my civilian job.
I first heard about Winget a few years ago, but it has always seemed "Latin" to me, despite the fact that all the machines mentioned before were command-based in the ancient times!
I got really close to it with WingetUI, so I thought I'd make the interface available to my compatriots in our mother tongue.
I wish good work to all contributors!
